lib/swaps/Swaps.ts

```diff
@@ -84,19 +70,40 @@ class Swaps extends EventEmitter {
   }
 
   /**
-   * Calculates the amount of subunits/satoshis each side of a swap should receive.
+   * Calculates the currencies and amounts of subunits/satoshis each side of a swap should receive.
    * @param quantity The quantity being swapped
    * @param price The price for the swap
    * @param isBuy Whether the maker order in the swap is a buy
-   * @returns An object with the calculated `makerAmount` and `takerAmount` values
+   * @returns An object with the calculated maker and taker values.
    */
-  public static calculateSwapAmounts = (quantity: number, price: number, isBuy: boolean, pairId: string) => {
+  private static calculateMakerTakerAmounts = (quantity: number, price: number, isBuy: boolean, pairId: string) => {
+    const { inboundCurrency, inboundAmount, outboundCurrency, outboundAmount } =
+      Swaps.calculateInboundOutboundAmounts(quantity, price, isBuy, pairId);
+    return {
+      makerCurrency: inboundCurrency,
+      makerAmount: inboundAmount,
+      takerCurrency: outboundCurrency,
+      takerAmount: outboundAmount,
+    };
+  }
+
+  /**
+   * Calculates the incoming and outgoing currencies and amounts of subunits/satoshis for an order if it is swapped.
+   * @param quantity The quantity of the order
+   * @param price The price of the order
+   * @param isBuy Whether the order is a buy
+   * @returns An object with the calculated incoming and outgoing values.
+   */
+  public static calculateInboundOutboundAmounts = (quantity: number, price: number, isBuy: boolean, pairId: string) => {
```

Comment:
I really like that it's returning `inboundAmount/outboundAmount` now. Much more clearer than `makerAmount/takerAmount`.